src/components/profiles-modals/ImportProfileModal.vue

```diff
@@ -317,32 +306,83 @@ export default class ImportProfileModal extends mixins(ProfilesMixin) {
             <h2 class="modal-title">Packages to be installed</h2>
         </template>
         <template v-slot:body>
-            <OnlineModList :paged-mod-list="profileToOnlineMods" :read-only="true" />
+            <OnlineModList :paged-mod-list="profileMods.known" :read-only="true" />
+            <div v-if="!profileMods.known.length || profileMods.unknown.length" class="notification is-warning margin-top">
+                <p v-if="!profileMods.known.length">
+                    None of the packages in the profile were found on Thunderstore:
+                </p>
+                <p v-else>
+                    Some of the packages in the profile were not found on Thunderstore:
+                </p>
+
+                <p class="margin-top">{{ profileMods.unknown.join(", ") }}</p>
+
+                <p v-if="profileMods.known.length" class="margin-top">
+                    These packages will not be installed.
+                </p>
+                <p v-else class="margin-top">
+                    Ensure the profile is intended for the currently selected game.
+                </p>
+
+                <p class="margin-top">
+                    Updating the mod list from Thunderstore might solve this issue.
+
+                    <span v-if="$store.state.tsMods.modsLastUpdated">
+                        The mod list was last updated on {{ valueToReadableDate($store.state.tsMods.modsLastUpdated) }}.
+                    </span>
+
+                    <br />
+
+                    <span v-if="$store.state.tsMods.isThunderstoreModListUpdateInProgress">
+                        {{ $store.state.tsMods.thunderstoreModListUpdateStatus }}
+                    </span>
+                    <span v-else-if="$store.state.tsMods.thunderstoreModListUpdateError">
+                        Error updating the mod list:
+                        {{ $store.state.tsMods.thunderstoreModListUpdateError }}.
+                        <a @click="$store.dispatch('tsMods/fetchAndProcessPackageList')">Retry</a>?
+                    </span>
+                    <span v-else>
+                        Would you like to
+                        <a @click="$store.dispatch('tsMods/fetchAndProcessPackageList')">update now</a>?
+                    </span>
+                </p>
+            </div>
         </template>
         <template v-slot:footer>
-            <button
-                id="modal-review-confirmed"
-                class="button is-info"
-                @click="onProfileReviewConfirmed();">
-                Import
-            </button>
+            <div class="container is-flex is-justify-content-space-between">
+                <button
+                    id="modal-no-mods-to-import"
+                    class="button is-danger"
+                    @click="closeModal"
+                >
+                    Cancel import
```

Comment:
I think `Cancel` alone is fine? Not overly fussed.

We don't usually offer "Cancel" actions though. It's expected that the user would just click off the modal. Might be nice to make it consistent in other places if we're going to start doing this.

Reply:
🤷🏻